src/comps/product-listing/productListing.jsx

```diff
@@ -0,0 +1,73 @@
+import "./productListing.css"
+
+
+import { useProduct} from "./../../context/products.context.jsx"
+
+
+const ProductListing = ()  => {
+
+    const {itemList, setItemList} = useProduct()
+
+    return (
+
+ 
+    <main className="product-display-container">
+
+
+        <div className="product-display-component">
+            <div className="product-grid">
+             
+
+              
+
+                { itemList.map( ({title , seller ,price,categoryName,image,rating}) => 
+                { 
+                    return(
+                        <div className="ecomm vertical-card">
+                    <div className="vertical-card-image">
+                        <img
+                            src={image}
+                            alt="card-pic" />
+                    </div><div className="vertical-badge">{rating} ⭐</div><button className="close-btn">
+                            <i className="fi-xwluxl-heart-wide"></i>
+                        </button><div className="card-heading">{title}</div>
+                        <div className="card-text">Sold By:{seller}</div>
+                        <div className="card-price">₹.{price}</div>
+                        <button className="btn-cart btn">ADD TO CART</button>
+                        </div>
+         
+         )} )} 
+
+
+    
+
+
+                <div className="ecomm vertical-card">
+                    <div className="overlay-layer">
```

Comment:
you can create a separate component for vertical card as well to make it reusable and cleaner code